packages/terra-dropdown-button/src/DropdownButton.module.scss

```
@@ -0,0 +1,60 @@
+@import '~terra-mixins/lib/Mixins';
+
+@import './variables';
+@import './mixins';
+
+:local {
+  .dropdown-button-type {
+    border-radius: var(--terra-dropdown-button-border-radius, 3px);
```

Update for consistency with other variable names
```suggestion
    border-radius: var(--terra-dropdown-button-dropdown-type-border-radius, 3px);
```